install-steps.md

```diff
@@ -1,193 +1,821 @@
+NOTE: Ensure to change the appropriate variables to match your environment.
+
+2. Create a directory to store cluster configs
+    ~~~sh
+    mkdir ~/clusterconfigs
+    cp install-config.yaml ~/clusterconfigs
+    ~~~
+
+3. Ensure all baremetal nodes are powered off prior to installing the OpenShift cluster
+   ~~~sh
+   ipmitool -I lanplus -U <user> -P <password> -H <management-server-ip> -p 6230 power off
+   ~~~
+
+4. Run the OpenShift Installer
+    ~~~sh
+    ./openshift-baremetal-install --dir ~/clusterconfigs --log-level debug create cluster
+    ~~~
+
+## Fix Metal3 ConfigMap
+
+With the installation complete. The focus shifts to fixing the metal3 ConfigMap
+in order to deploy new nodes. 
+
+Due to this [issue](https://github.com/openshift/installer/pull/2449) we need 
+to fix the ConfigMap for the Metal3 operator.
+
+This ConfigMap is used to notify `ironic` how to PXE boot new nodes.
+
+1. Create the sample ConfigMap `metal3-config.yaml.sample`
+   ~~~yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: metal3-config
+      namespace: openshift-machine-api
+    data:
+      cache_url: ${RHCOS_URI}
+      deploy_kernel_url: http://172.22.0.3:6180/images/ironic-python-agent.kernel
+      deploy_ramdisk_url: http://172.22.0.3:6180/images/ironic-python-agent.initramfs
+      dhcp_range: 172.22.0.10,172.22.0.100
+      http_port: "6180"
+      ironic_endpoint: http://172.22.0.3:6385/v1/
+      ironic_inspector_endpoint: http://172.22.0.3:5050/v1/
+      provisioning_interface: <NIC1>
+      provisioning_ip: 172.22.0.3/24
+      rhcos_image_url: ${RHCOS_PATH}${RHCOS_URI}
+    ~~~
+    NOTE: The `provision_ip` should be modified to an available IP on the `provision` network. The default is `172.22.0.3`
+
+2. Create the final ConfigMap
+    ~~~sh
+    export COMMIT_ID=$(./openshift-baremetal-install version | grep '^built from commit' | awk '{print $4}')
+    export RHCOS_URI=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json | jq .images.openstack.path | sed 's/"//g')
+    export RHCOS_PATH=$(curl -s -S https://raw.githubusercontent.com/openshift/installer/$COMMIT_ID/data/data/rhcos.json | jq .baseURI | sed 's/"//g')
+    envsubst < metal3-config.yaml.sample > metal3-config.yaml
+    ~~~
+3. Load the ConfigMap
+    ~~~sh
+    ./oc create -f metal3-config.yaml
+    ~~~
+
+## Backup Cluster Config
+
+At this point you will have a working OpenShift 4 cluster on baremetal. In order
+to take advantage of the baremetal hardware that was the provision node, we will
+repurpose the provisioning node as a worker. Prior to re-provisioning 
+the node, it is recommended to backup some existing files.
+
+~~~sh
+# Tar the cluster configs folder and download it to your laptop
+tar cvfz clusterconfig.tar.gz ~/clusterconfig
+# Copy the Private part for the SSH Key configured on the install-config.yaml to your laptop
+tar cvfz clusterconfigsh.tar.gz ~/.ssh/id_rsa*
+~~~
+
+# Deploy Worker Node
```

Comment:
i would deploy the worker during the initial installation, by setting them properly in the install-config.yaml

Reply:
I actually need to change the title. This doc already deploys workers via install-config.yaml . This is to convert the provision node into a worker node after initial deployment.